icu4j/main/tests/core/src/com/ibm/icu/dev/test/timezone/TimeZoneTest.java

```diff
@@ -2390,6 +2390,29 @@ public void TestRawOffsetAndOffsetConsistency22041() {
                          zone.getRawOffset(), raw);
         }
     }
+
+    @Test
+    public void TestGetIanaID() {
+        final String UNKNOWN = "Etc/Unknown";
+
+        final String[][] TESTDATA = {
+                {"",                    UNKNOWN},
+                {null,                  UNKNOWN},
+                {"America/New_York",    "America/New_York"},
+                {"Asia/Calcutta",       "Asia/Kolkata"},
+                {"Europe/Kiev",         "Europe/Kyiv"},
+                {"Europe/Zaporozhye",   "Europe/Kyiv"},
+                {"Etc/GMT-1",           "Etc/GMT-1"},
+                {"Etc/GMT+20",          UNKNOWN},
+                {"PST8PDT",             "PST8PDT"},
+                {"GMT-08:00",           UNKNOWN},
+        };
```

Comment:
we should also check the case that the input is NKNOWN

Reply:
done